keyboards/wolfmarkclub/wm1/bootloader_defs.h

```diff
@@ -0,0 +1,10 @@
+/* Address for jumping to bootloader on STM32 chips. */
+/* It is chip dependent, the correct number can be looked up here (page 175):
+ * http://www.st.com/web/en/resource/technical/document/application_note/CD00167594.pdf
+ * This also requires a patch to chibios:
+ *  <tmk_dir>/tmk_core/tool/chibios/ch-bootloader-jump.patch
+ */
+
+// STM32F103* does NOT have an USB bootloader in ROM (only serial),
+//  so setting anything here does not make much sense
+#define STM32_BOOTLOADER_ADDRESS 0x80000000
```

Comment:
```suggestion
#define STM32_BOOTLOADER_ADDRESS 0x08000000
```

Can we make this change, but also do a quick test to add `RESET` to a keymap to see if it'll actually enter bootloader? The stock bootloader will be at `0x08000000` so if we set it up that we perhaps we can ensure the keycode works too.

Reply:
I will ignore your comment

Reply:
Lets just ignore this for now, unless you're holding down ESC during bootup the bootloader will end up executing QMK anyway.